packages/dcos-integration-test/extra/test_tweeter_demo.py

```diff
@@ -0,0 +1,86 @@
+import os
+import time
+
+import requests
+
+from test_packaging import (
+    _skipif_insufficient_resources
+)
+
+DEMO_RESOURCE_REQUIREMENTS = {
+    'number_of_nodes': 5,
+    'node': {
+        'disk': 0,
+        'mem': 0,
+        'cpus': 1
+    }
+}
+
+
+def _get_tweeter_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/tweeter.json')
+    return response.json()
+
+
+def _get_post_tweets_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/post-tweets.json')
+    return response.json()
```

Comment:
Feel free to ignore this philosophical point: Any reason to not just do this?
```
return requests.get(
    'http://...').json()
```
E.G. should it be an idiom that we limit the number of function calls to one per line?

Reply:
Hm, I don't think either is wrong/right for something this simple. I like having one action per line even for common things like this because I have a simple mind :P I suppose you could argue that the git diff is better in the this case if there are two lines and you changed the URL, because then you wouldn't have to look to see if it was the formatting action or the get action that was changed 🤓

Reply:
I prefer the existing one. "get the response" object and get the json() from it. The idea is, in future  you could insert something between line 32 and 33.